1.14/test/run

```diff
@@ -144,6 +146,31 @@ test_command() {
   return 0
 }
 
+test_logs() {
+  local expected="$1"
+  local message="$2"
+  local logfile="$3"
+
+  if [ "$message" ]; then
+    info ${message}
+  fi
+  if [ -n "$logfile" ]; then
```

Comment:
Had to remove this branch since it would try to pick up `stdout` or `stderr` files from the containers cwd and there were none. Is this what is expected to happen?